INeedCoffee/sort.rb

```diff
@@ -0,0 +1,14 @@
+class Sort
+  def self.DistanceBetweenCoffeeShopAndUser(coffeeShopName, distanceFromUserToCoffeeShop, sortedDistanceFromUserToCoffeeShop)
```

Comment:
Pluralize the names accordingly. Search more on how to implement such logic in Ruby.